apps/expo/src/app/contact-us/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Linking, Text, View } from 'react-native'
+
+import { Feather, Fontisto, SimpleLineIcons } from '@expo/vector-icons'
+
+import NavBar from '~/components/ui/nav-bar/NavBar'
+
+export default function ContactUsPage() {
+  return (
+    <View style={{ flex: 1, backgroundColor: '#1C1B1B' }}>
+      {/* Header */}
+      <NavBar center='Contact Us' />
+
+      <View style={{ flex: 1, paddingHorizontal: 20, paddingTop: 20 }}>
```

Comment:
Have you tried using tailwind styles for these view elements? I think they work on `View`. Same with the `Text` elements.

Additionally, to reduce the amount of styles vomit (not your fault, just the nature of mobile dev I guess), you could take repeated styles and save them to variables outside of the component and then reference them in the component body. For example:

```ts
const viewStyles = { flexDirection: 'row', alignItems: 'center', marginTop: 20 };
const textStyles = { fontSize: 18, color: '#CACACA' }

then where you use the View and Text elements:

<View style={viewStyles}>...</View>
<Text style={textStyles}>...</Text>
```

But of course, if you're able to use tailwind styles, it would be more like:
```ts
const viewStyles="tailwind styles here"

<View className={viewStyles}>...</View>
```

I believe in your victory, soldier!